src/argon2.lua

```diff
@@ -63,6 +61,9 @@ local _M     = {
     _URL     = "https://github.com/thibaultCha/lua-argon2-ffi",
 }
 
+local encoded_len = function(salt_len, hash_len)
+    return 40 + (math.ceil(salt_len / 3) + math.ceil(hash_len / 3)) * 4
```

Comment:
Actually argon2 provide [argon2_encodedlen](https://github.com/P-H-C/phc-winner-argon2/blob/master/include/argon2.h#L427) returns the encoded hash length for the given input parameters. 

I will declare this function in `ffi.cdef`, instead of that formula.